css/main.css

```diff
@@ -198,13 +213,14 @@ textarea {
 }
 
 .about__info__description {
-  font-size: 1.563em;
+  font-size: 1.4em;
   letter-spacing: 0.5px;
-  line-height: 2.250em;
+  line-height: 1.7em;
 }
 
 .about__details {
   display: flex;
+  margin-top: 40px;
```

Comment:
Instead of having to do this, use `display: flex` on `.about__info` with `flex-direction: column` and then `flex: 1` on the `.about__info__description`.